src/version.ts

```diff
@@ -0,0 +1 @@
+export const LIB_VERSION = "0.6.0";
```

Comment:
Probably we can get it from package.json.

Reply:
This relates to what I was talking about in Slack. If I import package.json, then it messes up how sources are placed in the `lib` folder. For transparency this is the Slack message:

>  I figured out why. Apparently if you import any file outside of src, then tsc will maintain the src directory for you. So when I did this:
`import sdkPackage from '../../package.json';`
It made so that package.json was copied to `lib` and hence the source code had to go into `src`. Which makes sense, but came as a bit of surprise.

Reply:
How about `JSON.parse(path.join(__dirname, "..", "package.json"))`?

Reply:
@miraclx couldn't use your snippet as `__dirname` is not supported in ES modules. Went with `JSON.parse(fs.readFileSync(path.join(fileURLToPath(import.meta.url), "..", "..", "package.json"), 'utf-8'));` as the next best option.

Reply:
PS: this is equivalent to

`JSON.parse(fs.readFileSync(fileURLToPath(new URL("../package.json", import.meta.url)), "utf-8"))`. 😄

Reply:
This wouldn't work on Windows though, right?

Reply:
Tested, it works. `fileUrlToPath` makes the conversion.

Reply:
Ah, nice! I will change this over then